### PR TITLE
added simplified sail controller model and unit tests

### PR DESCRIPTION
**Pull request by Tsuchijo**
Adding Simplified Sail Controller and its unit tests

### CODE REVIEW

File: SailingYachtModel/getSailAngleSimple.m [hunk 1]
```diff
@@ -0,0 +1,25 @@
+function [sailAngle] = getSailAngleSimple(windAngle)
+%Simplified sail controller, approximates sail angle with basic linear
+%function
+%lower bound of stepwise function
+x1 = 0.25 * 3.14;
```
Comment (brucecui97) [inline SailingYachtModel/getSailAngleSimple.m]: nit: can we define 3.14 as a constant (e.g. PI)

File: SailingYachtModel/getSailAngleSimple.m [hunk 2]
```diff
@@ -0,0 +1,25 @@
+function [sailAngle] = getSailAngleSimple(windAngle)
```
Comment (brucecui97) [inline SailingYachtModel/getSailAngleSimple.m]: can you please add the link of the paper and the page reference as comment to this method? thanks

Comment (brucecui97) [inline SailingYachtModel/getSailAngleSimple.m]: think "apparentWindAngleRad" is a more descriptive name

Comment (brucecui97) [reply SailingYachtModel/getSailAngleSimple.m]: not sure if it's possible, but a brief description of the sail control law is helpful too, so people don't need to refer to the paper if they just want a high level understanding of the control law

File: SailingYachtModel/getSailAngleSimple.m [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+%Simplified sail controller, approximates sail angle with basic linear
+%function
+%lower bound of stepwise function
+x1 = 0.25 * 3.14;
+%upperbound of stepwise function
+x2 = (0.75) * 3.14;
+
+%minimum sail angle
+min = 0;
```
Comment (brucecui97) [inline SailingYachtModel/getSailAngleSimple.m]: "minSailAngle" and "maxSailAngle"

File: SailingYachtModel/getSailAngleSimple.m (hunk 2 above)
Comment (brucecui97) [reply SailingYachtModel/getSailAngleSimple.m]: nit: think "getSailAngleRadSimple" might be more descriptive. This is just my personal preference to help avoid unit conversion errors

**Review by brucecui97 — CHANGES_REQUESTED** [review]

File: SailingYachtModel/getSailAngleRadSimple.m [hunk 4]
```diff
@@ -0,0 +1,31 @@
+function [sailAngle] = getSailAngleRadSimple(apparentWindAngleRad)
+%Simplified sail controller, approximates sail angle with basic linear
```
Comment (brucecui97) [inline SailingYachtModel/getSailAngleRadSimple.m]: is linear same as stepwise?

File: SailingYachtModel/getSailAngleRadSimple.m [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+%function
+%Aprroximates Sail Control with a stepwise function, if angle to wind is
+%below x1 then sets sail angle to maxSailAngle, if the angle is above x2
+%then sets sail angle to minSailAngle, otherwise approximates sail angle
+%using a linear interpolations
+%Method taken from this paper:
+%https://core.ac.uk/download/pdf/79618904.pdf
+%lower bound of stepwise function
```
Comment (brucecui97) [inline SailingYachtModel/getSailAngleRadSimple.m:12]: nit: let's add an empty line above here

File: SailingYachtModel/getSailAngleRadSimple.m [hunk 6]
```diff
@@ -0,0 +1,31 @@
+function [sailAngle] = getSailAngleRadSimple(apparentWindAngleRad)
```
Comment (brucecui97) [inline SailingYachtModel/getSailAngleRadSimple.m:1]: let's wrap apparentWindAngleRad to either -pi to pi or 0 to 2pi just to be safe

File: SailingYachtModel/getSailAngleSimpleTest.m [hunk 7]
```diff
@@ -0,0 +1,4 @@
+fplot(@(z) getSailAngleRadSimple(z), [-pi, pi]);
```
Comment (brucecui97) [inline SailingYachtModel/getSailAngleSimpleTest.m]: let's add edge case tests as well. Also add a test that has angle outside -pi to pi or 0 to 2pi. E.g. 10pi just to catch bugs

**Review by brucecui97 — APPROVED** [review]
